package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maplibre-gl-js-amplify",
-  "version": "1.2.9-geo",
```

Comment:
Just want to make sure that this is the right version to revert to?

Reply:
yep the verison in package.json isn't too important since semantic release is controlling the version number!